test-cases/tests/installation/a34-verify-sku-values.md

```diff
@@ -0,0 +1,35 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-fresh-install
+      - osd-post-upgrade
+    targets:
+      - 1.4.0
+tags:
+  - automated
+---
+
+# A34 - Verify SKU values
+
+## Prerequisites
+
+- Logged in to a testing cluster as a kubeadmin
+
+## Steps
```

Comment:
Do we need to have manual steps if this is already an automated test?

Reply:
Well, I probably should have omitted the **bash commands** in the test case, because we don't expect this test case to be performed manually :)
But I would leave the **steps** there, in case someone would like to check how we are testing the feature without need to read it from the code of the automated test, if that makes sense

Reply:
yep, that makes sense. 👍